.github/workflows/pandas-tests.yaml

```diff
@@ -0,0 +1,27 @@
+name: Pandas Test Job
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+
+jobs:
+  pandas-tests:
+      # run the Pandas unit tests
+      secrets: inherit
+      uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-24.06
+      with:
+        matrix_filter: map(select(.ARCH == "amd64")) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(min_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
```

Comment:
Do we need to compare both CUDA 11 and CUDA 12? I think that's what this matrix is doing. Maybe we can do just CUDA 12. If anything I would expect results to vary with Python version but not CUDA version.

Reply:
Just one is fine, how do we do that?

Reply:
Pushed a change to achieve this.